data/common/items/items/susie_tea.lua

```diff
@@ -0,0 +1,90 @@
+local item = HealItem{
+    -- Item ID (optional, defaults to path)
+    id = "susie_tea",
+    -- Display name
+    name = "Susie Tea",
+    -- Custom variable for this item to determine the owner of the tea.
+    owner = "susie",
+
+    -- Item type (item, key, weapon, armor)
+    type = "item",
+    -- Item icon (for equipment)
+    icon = nil,
+
+    -- Battle description
+    effect = "Healing\nvaries",
+    -- Shop description
+    shop = "",
+    -- Menu description
+    description = "It's own-flavored tea.\nThe flavor just says \"Susie.\"",
+
+    -- Amount healed (HealItem variable)
+    heal_amount = 0,
+    -- Custom variable for this item, determines the healing value for each character.
+    heal_variants = {
+        ["kris"] = 120, 
+        ["susie"] = 10, 
+        ["ralsei"] = 120, 
+        ["noelle"] = 400,
+        ["thrash"] = 100,
+        ["default"] = 50
+    },
```

Comment:
same as changes to Kris Tea, default to `heal_amount`

```suggestion
    -- Amount healed (HealItem variable)
    heal_amount = 50,
    -- Custom variable for this item, determines the healing value for each character.
    heal_variants = {
        ["kris"] = 120, 
        ["susie"] = 10, 
        ["ralsei"] = 120, 
        ["noelle"] = 400
    },
```